myapp/app/models/user.rb

```diff
@@ -1,16 +1,25 @@
 class User < ApplicationRecord
   has_many :tasks
 
+  acts_as_paranoid
+
+  USER_MIN = 5
```

Comment:
[NITS] `MIN_USERNAME_LENGTH`, `MAX_USERNAME_LENGTH`: These names clearly indicate that they represent the minimum and maximum lengths for a username.

Reply:
addressed. thanks